yt_dlp/extractor/vbox7.py

```diff
@@ -29,19 +30,25 @@ class Vbox7IE(InfoExtractor):
             'timestamp': 1470982814,
             'upload_date': '20160812',
             'uploader': 'zdraveibulgaria',
-        },
-        'params': {
-            'proxy': '127.0.0.1:8118',
+            'view_count': int,
+            'duration': 2640,
         },
     }, {
         'url': 'http://vbox7.com/play:249bb972c2',
-        'md5': '99f65c0c9ef9b682b97313e052734c3f',
+        'md5': 'da1dd2eb245200cb86e6d09d43232116',
         'info_dict': {
             'id': '249bb972c2',
             'ext': 'mp4',
             'title': 'Смях! Чудо - чист за секунди - Скрита камера',
+            'uploader': 'svideteliat_ot_varshava',
+            'view_count': int,
+            'timestamp': 1360215023,
+            'thumbnail': 'https://i49.vbox7.com/design/iconci/png/noimg6.png',
```

Comment:
Really? `noimg6.png` isn't a likely name for a valid image. Should it be filtered out?
I get:
```
            'thumbnail': 'https://i49.vbox7.com/o/249/249bb972c20.jpg',
```

Reply:
I added this to https://github.com/yt-dlp/yt-dlp/pull/8968. I am not sure why the API responded like this when I wrote the extractor. Maybe the image took some time to be generated?
We will have to see how many of the videos remain in ~4 days, when the site will supposedly delete most user-generated content.